packages/core/src/compiler/Style.ts

```diff
@@ -2254,27 +2258,29 @@ const evalExprs = (
   trans: Translation
 ): Result<ArgVal<ad.Num>[], StyleDiagnostics> =>
   all(
-    args.map((expr) => evalExpr(mut, canvas, stages, { context, expr }, trans))
+    args.map((expr) => {
+      return evalExpr(mut, canvas, stages, { context, expr }, trans);
+    })
   ).mapErr(flatErrs);
 
-const argValues = (
-  mut: MutableContext,
-  canvas: Canvas,
-  stages: OptPipeline,
-  context: Context,
-  args: Expr<C>[],
-  trans: Translation
-): Result<(ShapeVal<ad.Num> | Value<ad.Num>)["contents"][], StyleDiagnostics> =>
-  evalExprs(mut, canvas, stages, context, args, trans).map((argVals) =>
-    argVals.map((arg) => {
-      switch (arg.tag) {
-        case "ShapeVal": // strip the `ShapeVal` tag
-          return arg.contents;
-        case "Val": // strip both `Val` and type annotation like `FloatV`
-          return arg.contents.contents;
-      }
-    })
-  );
+// const argValues = (
```

Comment:
Is this still useful? If not, delete these lines.

Reply:
Sure will do this